config.lua

```diff
@@ -6,13 +6,13 @@ box.cfg{
 
```

Comment:
Please rebase to master branch and add changelog entry.

Reply:
Done.